docker-service/main.tf

```diff
@@ -11,6 +11,7 @@ module "service_container" {
   mounts                 = var.service.mounts
   container_capabilities = var.service.capabilities
   commands               = var.service.commands
+  host_connection        = data.infisical_secrets.host_connections.secrets[var.service.host].value
```

Comment:
This right here is the line that's causing the issues....

Reply:
https://developer.hashicorp.com/terraform/language/modules/develop/providers

https://developer.hashicorp.com/terraform/language/modules/develop/providers#legacy-shared-modules-with-provider-configurations